static/js/zamboni/reviewers.js

```diff
@@ -242,6 +242,27 @@ function callReviewersAPI(apiUrl, method, data, successCallback) {
   });
 }
 
+$('#id_attachment_file').on('change', function () {
+  $('#attachment_errors').empty();
+  const file = this.files[0];
+  const max_upload_size = $(this).data('max-upload-size');
+  if (file) {
+    if (file.size > max_upload_size) {
+      error = $('<ul>')
+        .attr('class', 'errorlist')
+        .append(
+          $('<li>').append(
+            format(gettext('Your file exceeds the maximum size of {0}.'), [
+              '200MB',
```

Comment:
We don't have to hardcode this:
```
Intl.NumberFormat(document.documentElement.lang, {
  notation: 'compact',
  style: 'unit',
  unit: 'byte',
  unitDisplay: 'narrow',
}).format(max_upload_size);
```

(It gives 210MB because it's using SI units but that's fine)

Reply:
Wouldn't it be confusing for a 208MB file to reject with this error?

Reply:
It already works actually - I think our python code is using SI units, so your patch already allows 210MB (200 MiB) files server-side.